### DISCUSSION TITLE
Hide local albums from timeline

### DISCUSSION THREAD

**Discussion by nsg** [immich-app/immich#1236] [General]
Phone: Android, same behavior in the Emulator (I tried to figure out the answer myself, and failed)

Most other apps that I use only shows only the camera roll (`./DCIM/Camera`) on the timeline. Immich picks up all albums like screenshots, images from other various applications. My phone also saves and extra RAW image for each picture taken (outside the camera roll, in a separate folder). Immich picks up that as well, so I have always duplicate pictures in the timeline.

Is it possible to select what local albums/folders are shows in the timeline (under Photos)?

I'm missing something, doing something wrong or is this a feature still missing? Any workarounds?

**Comment by alextran1502** [comment]
The local timeline is the content of the album(s) you select for backup, so as long as you select the correct album you want for backup (include) and exclude the album(s) that you don't want to backup, the content on the timeline will be the one you want to be backup to Immich

**Reply by nsg** [reply]
Ah, I see. I haven't selected any albums to be backed up (I guess that default the timeline to everything?)... I'm already backing my phone up with Nextcloud and I intended to feed Immich pictures via the cli/api. No point to back things up twice.

**Reply by nsg** [reply]
Super, got it working, an excellent app! Even if I use it a little incorrectly I'm sure this has the possibility to solve a lot of problems for me and my girlfriend. Thank you for all your work.

Now I try to overengineer some form av contraption with the API to keep me entertained :grin:

**Comment by Antroxin** [comment]
Hello, I'm using IOs App Version 1.54.0 build .94 and other albums not selected as backup are showing in the timeline. It appears all my shared albums that contain repeated photos that I added from my library so they are shown twice in the timeline (once marked as backed up with ticked cloud, and the other as not in the server as a slashed cloud).

**Comment by emes81** [comment]
I think an option to do this via folder structure could be useful here. I have a few hundred music folders on my device, all with artwork; this means there are good few hundred folders to sift through before I can locate the DCIM folder, which is the only folder I want to upload. 
Selecting it simply through a folder hierarchy would be much more efficient.

**Reply by HikariNoKitsune** [reply]
I would love this. The ability to see your file structure in general would be nice, and the ability to just select a folder within that structure. 

I say this primarily because there are folders which are hidden, and thus not detected by Immich, that I would like to have backed up using Immich. For instance, an app that has its own image folder which it prefixes with a . (period) so that only it can view the contents. Being able to navigate to that specific folder and just tap the folder, selecting it for backup, would be awesome.
